docs/content/preview/yugabyte-platform/back-up-restore-universes/back-up-universe-data/ysql.md

```diff
@@ -37,23 +37,23 @@ showAsideToc: true
 
 You can use YugabyteDB Anywhere to back up your YugabyteDB universe YSQL data.
 
-To schedule backups for a later time or as a recurring task, see [Schedule universe YSQL data backups](../../schedule-data-backups/ysql/).
+To view, [restore](../../restore-universe-data/ysql/), or delete existing backups for your universe, navigate to that universe and select **Backups**, as per the following illustration:
```

Comment:
All the comments except the non-transactional one apply here too. There was never non-transactional backup support for YSQL.

Reply:
Okay